libs/design/src/molecules/list/list/list.component.ts

```diff
@@ -32,4 +51,36 @@ export class DaffListComponent {
   @HostBinding('class.daff-list--navigation') get navigation() {
     return this.mode === DaffListModeEnum.Navigation;
   }
+  
+  listType: DaffListType;
+
+  constructor(private elementRef: ElementRef, private renderer: Renderer2) {}
+
+  ngOnInit() {
+    for (const attr of LIST_HOST_ATTRIBUTES) {
```

Comment:
Given that `list-type` is necessarily one of these, can most of this code simply be...

```
get listType(): DaffListType  {
 return this._getHostElement().localName
}

Reply:
Updated